src/sentry/static/sentry/app/components/seenByList.jsx

```diff
@@ -53,8 +53,8 @@ export default class SeenByList extends React.Component {
     //       gracefully handing this case.
     //
     // See: https://github.com/getsentry/sentry/issues/2387
-
-    if (seenBy.length === 0) {
+    const displayUsers = (seenBy || []).filter(user => activeUser.id !== user.id);
```

Comment:
I suspect the comment above (and the need to do `|| []`) is  no longer valid since it looks like seenBy is always guaranteed to be an array